doc/dfu/quark_dfu_uart.rst

```diff
@@ -0,0 +1,140 @@
+Quark DFU over UART Guide
+#########################
+
+.. contents::
+
+Hardware Setup
+**************
+
+The hardware setup consists of two main steps:
+
+ * Connecting the board to a Linux host using UART.
+ * Having the possibility to connect the DM GPIO pin to GND or to press the DM
+   Button.
+
+The DM GPIO pin is the pin used to put the device into DM mode. The device has
+to be reset while the GPIO is connected to GND. On Quark Developer Boards a
+button connects the DM GPIO to GND if pressed.
+
+Software Setup
+**************
+
+Enabling DM functionality in ROM
+================================
+
+In order to use the Device Management (DM) functionality over UART on the target
+platform, you must install a ROM with such functionality enabled. To do that,
+perform the following steps:
+
+ * Compile the ROM code using the DM_ENABLE flag:
+   ::
+
+       make rom ENABLE_DM=1
```

Comment:
Can we add separate command for each ROM ?

make rom SOC=quark_se ENABLE_DM=1
make rom SOC=quark_d2000 ENABLE_DM=1